LayoutTests/imported/w3c/web-platform-tests/html/browsers/browsing-the-web/navigating-across-documents/source/navigate-child-function-parent-then-fragment-expected.txt

```diff
@@ -1 +1,9 @@
 
+
+Harness Error (TIMEOUT), message = null
+
+TIMEOUT
```

Comment:
I'm concerned to see this. How long does this test take to run with this change? It's not necessarily a problem if the test runs quickly, but taking 30 seconds to run a passing test is clearly bad.